pepper-apis/src/test/java/org/broadinstitute/ddp/route/GovernedParticipantRegistrationRouteTest.java

```diff
@@ -38,6 +45,21 @@ public static void setup() {
         url = RouteTestUtil.getTestingBaseUrl() + endpoint;
     }
 
+    @After
+    public void cleanup() {
```

Comment:
@zyme had reported that he saw the `UserGovernanceDaoTest` fail locally. I found that it was failing because we were not cleaning up test data thoroughly, so added cleanup here.